iocore/net/UnixNetVConnection.cc

```diff
@@ -1384,7 +1366,12 @@ UnixNetVConnection::free(EThread *t)
 {
   ink_release_assert(t == this_ethread());
 
+  // cancel OOB
+  cancel_OOB();
   // close socket fd
+  if (con.fd != NO_FD) {
+    NET_SUM_GLOBAL_DYN_STAT(net_connections_currently_open_stat, -1);
+  }
   con.close();
```

Comment:
According to `TS-4187: Keep the currently_open connections in sync with the number of sockets open`, move them closely.